src/features/advanced-content-insights/AdvancedContentInsightsToggle.tsx

```diff
@@ -45,7 +52,7 @@ const AdvancedContentInsightsToggle = ({
         <Toggle
             className="AdvancedContentInsightsToggle"
             data-testid="insights-toggle"
-            description={!hasTooltip && !isChecked && description}
+            description={description}
```

Comment:
would it work to keep:
`const description = <FormattedMessage ...` ~line 28
and
`<Tooltip text={description}>` ~line 42
the way it was previously and do:
`description={hasDescription && description}`
instead?

Reply:
did that in anticipation that product will want to change text and possibly to have different messages. but for now message can be shared between tooltip and description